### ISSUE TITLE
Problem in Adding The code to github 

### ISSUE THREAD

**Issue by ZeinaDaghlas** [sallyishak/SoftwareConstructionProject#3]
Searching for a solution must be found before 2-Jan-2016

**Comment by remalsaad** [comment]
Yes Zaina, thanks for adding this issue ,I'm searching for a solution about adding the code to github,I will apdate u .

**Comment by remalsaad** [comment]
Please all see this video https://www.youtube.com/watch?v=GCBROgPCox0

**Comment by remalsaad** [comment]
Dear,I'm installing egit as an plugin  in eclipes

**Comment by remalsaad** [comment]
Hi girls ,I have find the solution https://www.youtube.com/watch?v=r5C6yXNaSGo

**Comment by ZeinaDaghlas** [comment]
Thank you Remal , it so helpful .. am working on it on my laptop
